README.md

```diff
@@ -7,11 +7,21 @@ this repository is used to centralize all the proposals in one place
 apart from bug reports.
 
 Proposals are made by opening issues in this repository, which will
-then be discussed with fellow Godot users and contributors. Once a
+then be discussed with fellow Godot users and contributors. If a
 proposal is considered good to implement, it will be approved by a
 core developer.
 
-#### Rules for Submitting a Proposal
+Please note: making a proposal does not guarantee that your proposed
+feature will be accepted into the engine. In fact, in most cases your
+proposal will not be accepted. The Godot organization has very few
+resources and relies almost completely on irregular contributors to
+implement anything. Technical debt and software bloat accumulate
+quickly, as a result core developers have to be very selective about
+what features make it into the engine. For a deeper discussion on what
+features will be accepted see the section **How core developers
+evaluate proposals** below.
```

Comment:
While this is good to mention, I think it might be a bit *too* negative.

It's also important to keep in mind that the idea behind the proposals / GIP workflow is to outline technical proposals that can be implemented by any contributor with the blessing of the core team -- so resources available to current core contributors are only part of the equation - it's fine to have an approved but implemented proposal taking dust for a while until a new contributor that needs it comes up to implement it.

Reply:
I agree. However, I also think that setting expectations is important. Do you think it could be rephrased in a better tone, or would you like to cut this paragraph entirely?

Reply:
> Please note: making a proposal does not guarantee that your proposed
feature will be accepted into the engine.

That's the definition of a proposal: it may be accepted, or it may be not, I'd remove this. (again nobody is forcing anyone to do anything, even if one thinks that "someone has to do this").

> In fact, in most cases your proposal will not be accepted.

That's not a fact. Some might have a good eye on what features are suited for core. But I guess it's safe to say that the overall likelihood of proposed features being implemented is very low, but again if a person follows the evaluation guideline, the likelihood of a feature being implemented should be very high, unless the discretion of core developers takes higher priority. So I'd replace `your` with a more general statement using statistics, making it less personal. Something like this:
```
Bare in mind that most proposals may not end up being implemented directly in the engine.
```

That also tells how a proposed feature can be a good candidate for a plugin if a proposal is rejected for core, and becomes in alignment with `Technical debt and software bloat` statements.

I'm not yet sure whether the guideline is written for people *making* proposals, or for those *evaluating* proposals, but I guess both. If that's the case, any personal references such as `you` or `your` do not apply.

Reply:
Nitpick: Bear not bare in mind.

Reply:
I have cut the paragraph entirely. Better to remain positive.